corehq/apps/export/static/export/js/export_list.js

```diff
@@ -434,6 +467,56 @@ hqDefine("export/js/export_list", [
             return true;
         };
 
+        self.multiple = ko.computed(function () {
+            if (self.bulkDeleteList().length > 1) { return "s"; }
```

Comment:
Must've missed the first comment but I'll have it reference the colors from the .less file in another commit. 

Got rid of self.multiple and self.plural for a single variable self.isMultiple and defined separate blocks of text instead of trying to append a single 's' or change from 'this' to 'these'